apps/fabric-website/src/pages/ResourcesPage/ResourcesPage.tsx

```diff
@@ -54,6 +54,12 @@ export class ResourcesPage extends React.Component<any, any> {
                   <li><a className={ styles.relatedLink } href='https://static2.sharepointonline.com/files/fabric/fabric-website/files/sharepoint_toolkit.zip'>SharePoint XD Toolkit</a></li>
                 </ul>
 
+                <h3>Office Add-ins Toolkit</h3>
```

Comment:
Since we have 'Design' in the other two titles, maybe this should read 'Office Add-ins Design Toolkit' ?